src/Xamarin.Android.Tools.AndroidSdk/Jdks/JdkLocations.MacOS.cs

```diff
@@ -21,8 +21,10 @@ static IEnumerable<string> GetUnixConfiguredJdkPaths (Action<TraceLevel, string>
 		{
 			var config = AndroidSdkUnix.GetUnixConfigFile (logger);
 			foreach (var java_sdk in config.Root.Elements ("java-sdk")) {
-				var path    = (string) java_sdk.Attribute ("path");
-				yield return path;
+				var path = (string) java_sdk.Attribute ("path");
```

Comment:
Nullable reference types are enabled, and the original code -- as well as this updated code -- is "disabling" NRT by asserting that `string` and not `string?` is returned.

Reply:
I opted for the minimal surgical fix to solve the problem since I'm in an unfamiliar codebase. I can leave more comprehensive fix options to the maintainers. Thanks!